pretixdroid/app/src/main/java/eu/pretix/pretixdroid/ui/EventinfoActivity.java

```diff
@@ -0,0 +1,338 @@
+package eu.pretix.pretixdroid.ui;
+
+import android.os.AsyncTask;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.ListView;
+import android.widget.TextView;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.LinkedList;
+
+import eu.pretix.pretixdroid.AppConfig;
+import eu.pretix.pretixdroid.R;
+import eu.pretix.pretixdroid.net.api.ApiException;
+import eu.pretix.pretixdroid.net.api.PretixApi;
+
+/**
+ * This class is the activity for the Eventinfo page to let the user see statistics about their
+ * event.
+ * @author jfwiebe
+ */
+public class EventinfoActivity extends AppCompatActivity {
+    private AppConfig config;
+    private PretixApi api;
+
+    private ListView mListView;
+    private EventItemAdapter mAdapter;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_eventinfo);
+
+        this.mListView = (ListView)findViewById(R.id.eventinfo_list);
```

Comment:
Could you have Android Studio auto-format your code? That should insert a space after the first ) and also some other things.